tests/cases/fourslash/gotoDefinitionLinkTag1.ts

```diff
@@ -0,0 +1,48 @@
+///<reference path="fourslash.ts" />
```

Comment:
Can these tests handle GTD across files.  Would that be interesting to test?

Reply:
across files: yes; interesting: sort of -- the useful test is whether the span is right, not whether goto-def works across files. The unit tests show this by inspection, but adding a baseline is not a bad idea.